pmd-vm/src/main/java/net/sourceforge/pmd/lang/vm/ast/VmTokenManager.java

```diff
@@ -1,22 +1,21 @@
-/**
+/*
  * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
  */
 
-package net.sourceforge.pmd.lang.vm;
+package net.sourceforge.pmd.lang.vm.ast;
```

Comment:
It seems we moved the JavaTokenManager as well from `lang.java` to sub-package `lang.java.ast` - without noting this incompatibility...

Reply:
I think, it makes sense to internalize the VmTokenManager as well on master - the access should be through LanguageVersionHandler#getParser#getTokenManager and the interface TokenManager should be used.